File: observer/Samuel Sales/DesafioObserver_Samuel/src/Revistas/SuperInteressante.java
package Revistas;

import agencia.Agencia;

/**
 *
 * @author Samuel Sales - Facul
 */
public class SuperInteressante implements ObserverEditoraAbril {

    @Override
    public void atualizar(Agencia entrada) {
        System.out.println(entrada.getNome() + ": Revista SuperInteressante");
    }

}

File: observer/PadraoObserverII/PadraoObserverII/Revistas/SuperInteressante.cs
using PadraoObserverII.Interface;
using System;

namespace PadraoObserverII.Revistas
{
    class SuperInteressante : IObservar
    {
        public void Atualizar(Agencia entrada)
        {
            Console.WriteLine(entrada.Nome + ": Notícias da Revista Super Interessante!");
        }
    }
}
